doc/1.4/running-dmlc.md

```diff
@@ -17,8 +17,9 @@ The DML compiler and its libraries are available as part of the
 The `dmlc` compiler can be build locally. This requires an
 installation of the Simics 6 base package.
 
-In order to build the compiler, you can copy source directory `dmlc`
-into the `modules` subdirectory of your Simics project. The compiler
+In order to build the compiler, checkout [the DML repository
+](https://github.com/intel/device-modeling-language) into the
```

Comment:
I would remove the whitespace before `]`, though I don't think this is enforced by a style guide